launch/start.launch

```diff
@@ -0,0 +1,39 @@
+<launch>
+  
+  <node pkg="polhemus" type="polhemus_tf_broadcaster" name="polhemus_tf_broadcaster"/>
+
+  <!--Set output hemisphere parameters-->
+  <param name="x_hs" value="0" type="double"/>
+  <param name="y_hs" value="0" type="double"/>
+  <param name="z_hs" value="-1" type="double"/>
+
+  <!--Setup static transform polhemus_base <-> world-->
+  <node pkg="tf" type="static_transform_publisher" name="polhemus_world_broadcaster" args="0 0 0.54 0 0 3.1415926 polhemus_world polhemus_base 100"/>
```

Comment:
This should be in application-specific launch files, but good to leave as an example